### DISCUSSION TITLE
谢谢楼主，以前用的都是假绳网，现在终于用上真·绳网了

### DISCUSSION THREAD

**Discussion by ASTOU2311** [share121/inter-knot#4555] [灌水]
谢谢楼主，以前用的都是假绳网，现在终于用上真·绳网了

_Originally posted by @dalizi2333333 in https://github.com/share121/inter-knot/discussions/1898#discussioncomment-10372087_

**Comment by AetherLude** [comment]
请注意，本页面是issue追踪器（Issue tracker）而不是讨论区（Discussions）。绳网发帖请移步[讨论区](https://github.com/share121/inter-knot/discussions)。
请不要在此处灌水。您在此处的发帖不计入绳网渲染范围。

**Comment by rong-xiaoli** [comment]
> 请注意，本页面是issue追踪器（Issue tracker）而不是讨论区（Discussions）。绳网发帖请移步[讨论区](https://github.com/share121/inter-knot/discussions)。 请不要在此处灌水。您在此处的发帖不计入绳网渲染范围。

合理怀疑抄的我的issue template（bushi）
建议楼主close该issue
